.github/workflows/ci.yml

```diff
@@ -88,6 +88,9 @@ jobs:
             run: |
                 curl -L -O https://tiker.net/ci-support-v0
                 . ./ci-support-v0
+                if [[ "$DOWNSTREAM_PROJECT" == "pytential" ]]; then
```

Comment:
Could you add a conditional that makes this turn itself off on `main`? Since this and the pytential PR are mutually dependent, the process I like is the following:
- merge sumpy
- observe broken CI on sumpy `main`
- drop branch spec on pytential
- merge pytential
- rerun previously broken sumpy `main` CI, observe fixed